test/server/cards/05_LOF/bases/MysticMonastery.spec.ts

```diff
@@ -43,15 +43,15 @@ describe('Mystic Monastery\'s ability', function() {
             context.player1.clickCard(context.mysticMonastery);
             expect(context.player1.hasTheForce).toBe(true);
             expect(context.player2).toBeActivePlayer();
-            expect(context.getChatLogs(1)).toContain('player1 uses Mystic Monastery to gain the Force (1 use left)');
+            expect(context.getChatLogs(1)).toContain('player1 uses player1\'s base to gain the Force (1 use left)');
 
             context.player2.passAction();
             context.player1.setHasTheForce(false);
 
             context.player1.clickCard(context.mysticMonastery);
             expect(context.player1.hasTheForce).toBe(true);
             expect(context.player2).toBeActivePlayer();
-            expect(context.getChatLogs(1)).toContain('player1 uses Mystic Monastery to gain the Force (0 uses left)');
+            expect(context.getChatLogs(1)).toContain('player1 uses player1\'s base to gain the Force (0 uses left)');
```

Comment:
> player1 uses player1\'s base

Would it be feasible for these types of phrases to instead say `player1 uses their base`

Reply:
Were you suggesting this instead of showing the name of the base or would you do it everywhere? I refactor the implementation to keep the name of the base and it would be fairly easy to change `{0}'s base` to `their base`/`the opponent base`.